project/roadmap.md

```diff
@@ -4,61 +4,71 @@
-* [Mbed TLS] [Mbed TLS3.2](https://github.com/orgs/Mbed-TLS/projects/1#column-18338314)
-* [Mbed TLS] [TLS1.3 PSK](https://github.com/orgs/Mbed-TLS/projects/1#column-17950145)
-* [Mbed TLS] [TLS/X.509​ - Use PSA Long term secret isolation](https://github.com/orgs/Mbed-TLS/projects/1#column-183383222)
-* [Mbed TLS] [Use PSA: code size optimisation via PSA_CRYPTO_CONFIG - part1: symmetric](https://github.com/orgs/Mbed-TLS/projects/1#column-18883163)
-* [Mbed TLS] [Connection ID DTLS1.2](https://github.com/Mbed-TLS/mbedtls/pull/6264)
-
-**2023 CQ1 (In Development)**
+**2023 CQ2 (In Development)**
+* [Mbed TLS] [Code size optimisation (driver only build - ECC - Initial)](https://github.com/orgs/Mbed-TLS/projects/1#column-18986128)
+* [PSA Crypto] Memory Optimizations - Phase1
+* [PSA Crypto] PSA driver – Handle Opaque Persistent Key in Secure Element - Investigation
 * [Bignum] [ECP Curves field reduction - NIST](https://github.com/orgs/Mbed-TLS/projects/1#column-17950163)
-* [Mbed TLS] [PKCS7](https://github.com/orgs/Mbed-TLS/projects/1#column-17950135)
-* [PSA Crypto] [PSA Crypto restartable sign message](https://github.com/orgs/Mbed-TLS/projects/1#column-18883250)
-* [PSA Crypto] [ECJ-PAKE PSA Implementation](https://github.com/orgs/Mbed-TLS/projects/1#column-18883296)
-* [Mbed TLS] [Open CI Enhancements](https://ci.trustedfirmware.org/view/Mbed-TLS) 
-* [Mbed TLS] [Code size optimisation (driver only build - hashes inc. HMAC)](https://github.com/orgs/Mbed-TLS/projects/1#column-19083975)
-* [PSA Crypto] [PBKDF2 PSA Implementation](https://github.com/orgs/Mbed-TLS/projects/1#column-18986190)
+* [Mbed TLS] [Use PSA: misc. gaps](https://github.com/orgs/Mbed-TLS/projects/1#column-18337954)
+* [PSA Crypto] [PBKDF2 PSA SW Implementation](https://github.com/orgs/Mbed-TLS/projects/1#column-18986190)
  
-**2023 CQ2**
- * [Mbed TLS] [Code size optimisation (driver only build - Cipher and AEAD)](https://github.com/orgs/Mbed-TLS/projects/1#column-19075367)
- * [Mbed TLS] [TLS1.3 early data](https://github.com/orgs/Mbed-TLS/projects/1#column-18972084) 
- * [PSA Crypto] Memory Optimizations
-
 **2023 CQ3**
- * [Mbed TLS] [Code size optimisation (driver only build - ECC)](https://github.com/orgs/Mbed-TLS/projects/1#column-18986128)
+* [Mbed TLS] [Code size optimisation (driver only build - ECC)](https://github.com/orgs/Mbed-TLS/projects/1#column-18986128)
+* [Mbed TLS] [TLS1.3 early data](https://github.com/orgs/Mbed-TLS/projects/1#column-18972084) 
+* [PSA Crypto] Memory Optimizations
+* PAKE API investigation for SPAKE2+ 
+* [PSA Crypto] [PBKDF2 PSA Driver Implementation](https://github.com/orgs/Mbed-TLS/projects/1#column-18986190)
+
+**2023 CQ4**
+* [Mbed TLS] [Code size optimisation (driver only build - Cipher and AEAD)](https://github.com/orgs/Mbed-TLS/projects/1#column-19075367)
+* [PSA Crypto] PSA driver – Handle Opaque Persistent Key in Secure Element - Investigation
```

Comment:
This line is repeated from Q2. Should it be removed, or should this be a different work item?